Library/Notifications.swift

```diff
@@ -1,15 +1,15 @@
 import Foundation
 
 public enum CurrentUserNotifications {
-  public static let showNotificationsDialog = "CurrentUserNotifications.showNotificationsDialog"
-  public static let projectSaved = "CurrentUserNotifications.projectSaved"
   public static let dataRequested = "CurrentUserNotifications.dataRequested"
+  public static let environmentChanged = "CurrentUserNotification.environmentChanged"
+  public static let localePreferencesChanged = "CurrentUserNotification.localePreferencesChanged"
```

Comment:
`localePreferencesChanged` is the only new notification. I just alphabetized this enum.